scalafmt-cli/src/main/scala/org/scalafmt/cli/Cli.scala

```diff
@@ -132,7 +132,8 @@ object Cli {
                      inputMethods: Seq[InputMethod],
                      msg: String): TermDisplay = {
     val termDisplay = new TermDisplay(
-      new OutputStreamWriter(options.common.err),
+      new OutputStreamWriter(
+        if (options.noStdErr) options.common.out else options.common.err),
```

Comment:
How about we add a method `CliOptions.info: OutputStream` that performs this logic?